src/styles/components/_tile.scss

```diff
@@ -45,3 +45,17 @@
 .tile--519x187 {
   @include aspect-ratio(519, 187);
 }
+
+
+.tile__caption {
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  margin: 1em;
```

Comment:
Should we use `em` or `px`? I've noticed that most browsers still have issues with animations being blurry if they land on non-whole pixel values (which can happen with `em`).

Reply:
Sounds good